ansible/roles/cyhy_commander/defaults/main.yml

```diff
@@ -1,6 +1,12 @@
 ---
 # defaults file for cyhy_commander
 
+# The maximum number of jobs to assign to each nessus host (vulnscanner)
+jobs_per_nessus_host: 16
+
+# The maximum number of jobs to assign to each nmap host (portscanner)
+jobs_per_nmap_host: 8
```

Comment:
Feel free to re-word as you see fit, but I think it's important to indicate where in the `commander.conf` these values are being used, since there are multiple sections in that file (`DEFAULT`, `production`, `purge`, etc.)
```suggestion
# The maximum number of jobs to assign to each nessus host (vulnscanner)
# in the "production" section of the commander configuration file
jobs_per_nessus_host: 16

# The maximum number of jobs to assign to each nmap host (portscanner)
# in the "production" section of the commander configuration file
jobs_per_nmap_host: 8
```

Reply:
Please see [2efa82e](https://github.com/cisagov/cyhy_amis/pull/669/commits/2efa82e74084583452692010324ac50822ec2c42).

Reply:
Lovely!